src/github.rs

```diff
@@ -480,6 +480,18 @@ impl Issue {
         Ok(comment)
     }
 
+    // returns an array of one element
+    pub async fn get_first_comment(&self, client: &GithubClient) -> anyhow::Result<Vec<Comment>> {
+        let comment_url = format!(
+            "{}/issues/{}/comments?page=1&per_page=1",
+            self.repository().url(),
+            self.number,
+        );
+        Ok(client
+            .json::<Vec<Comment>>(client.get(&comment_url))
+            .await?)
+    }
```

Comment:
not too happy about this but I don't know another to ask GitHub to "return the first comment of an issue"